packages/manager/modules/sms/src/sms/sms/compose/telecom-sms-sms-compose.html

```diff
@@ -318,18 +333,18 @@ <h1 data-translate="sms_sms_compose_title"></h1>
                                     <span data-translate="sms_sms_compose_no_stop_clause_label"></span>
                                 </label>
                             </div>
-                            <div data-ng-if="$ctrl.displaySenderCustomizationAdvice">
-                                <strong class="text-uppercase" data-translate="sms_sms_compose_message_advice_title"></strong>
-                                <p data-translate="sms_sms_compose_message_advice_info"></p>
-                                <p>
-                                    <span data-translate="sms_sms_compose_message_advice_description_1"></span>
-                                    <a href="#"
-                                       data-ui-sref="sms.service.senders"
-                                       data-translate="sms_sms_compose_message_advice_description_2">
-                                    </a>
-                                    <span data-translate="sms_sms_compose_message_advice_description_3"></span>
-                                </p>
-                            </div>
+                        </div>
+                        <div data-ng-if="$ctrl.isShortNumber()">
+                            <p data-translate="sms_sms_compose_message_commercial_description"></p>
+                            <label>
```

Comment:
Why having an empty `<label>` tag ?

Reply:
The <label> is for the style.

Reply:
`label` shouldn't be used for the only purpose of styling.

Reply:
I removed it/

Reply:
I'm sorry, I might have not expressed myself clearly. It is a good practice to associate a `label` with an `input` as the `label` will indicate the input's caption. It helps associating in this case the `span` value with the `input`. If you remove that the link between both is lost if I'm correct. This might not have a visual impact but it could have with screen readers. Please do keep that `label` :)

Reply:
It's been replaced by an oui-checkbox instead.